+bisos/+iteration/+step/convergence.m

```diff
@@ -0,0 +1,102 @@
+classdef convergence < bisos.iteration.Step
+   
+properties
+    type = 'convergence';
+    varout = [];
+    varin;
+    data_prev = true;
+    
+    levels;
+    
+    % default properties for convergence verification
+    ctol = 10^-9;
+    domain = [-1 1];
+end
+
+methods
+    function step = convergence(~, vars, varargin)
+        % new level set convergence setup
+        
+        for i = 1:length(vars)
+            if isa(vars{i}, 'char')
+                step.varin{length(step.varin)+1} = vars{i};
+            end
+        end
+        
+        
+        for i = 1:2:length(vars)
+           step.levels{length(step.levels)+1} = {vars{i} vars{i+1}};   
+        end
+        
+        
+        % set new options
+        if isempty(varargin)
+            return;
+        end
+        
+        for i=1:2:length(varargin{:})
+            step.(varargin{:}{i}) = varargin{:}{i+1};
+        end
+    end
+    
+    function [sol,info,stop] = run(step,prob,info,sol,symbols,assigns,options)
+        % Run objective step.
+        
+        if info.iter==1
+           stop = false;
+           return 
+        end
+    
+        info.converged = true;
+        for i = 1:length(step.levels)
+            
```

Comment:
please add some comments about what this loop is doing and/or aiming to do.

Reply:
Solved in commit "Update +bisos/+iteration/+step/convergence.m"